main.js

```diff
@@ -52,12 +45,24 @@ var getMobileInfo = new Promise(function(resolve, reject) {
 });
 
 var loadingPromises = [initFS, getMobileInfo];
+
 jars.forEach(function(jar) {
   loadingPromises.push(load(jar, "arraybuffer").then(function(data) {
     CLASSES.addPath(jar, data);
   }));
 });
 
+var packs = [
+  // "build/classes.jar.js.pack",
+  // "build/tests.jar.js.pack"
```

Comment:
Packing was an experiment to see how much faster we could load pre-jitted code. I'll remove it for now, we can bring it back if we need to.